Sources/SwiftSyntaxBuilder/README.md

```diff
@@ -1,30 +1,64 @@
-# SwiftSyntaxBuilder
+# Swift Syntax Builder
 
-Declarative and type-safe wrapper around SwiftSyntax.
+**Swift Syntax Builder** is an open-source package for generating Swift code in a declarative way.
 
-## Example Usage
+> Note: SwiftSyntaxBuilder is still in development, and the API is not guaranteed to
+> be stable. It's subject to change without warning.
 
 ```swift
-let sourceFile = SourceFile {
-  Import("SwiftSyntax")
-
-  Struct("ExampleStruct") {
-    Let("syntax", of: "Syntax")
-  }
+let source = SourceFile {
+  ImportDecl(path: "Foundation")
+  ImportDecl(path: "UIKit")
+  ClassDecl(classOrActorKeyword: .class, identifier: "SomeViewController", membersBuilder: {
+    VariableDecl(.let, name: "tableView", type: "UITableView")
+  })
 }
 
-let syntax = sourceFile.buildSyntax(format: format, leadingTrivia: .zero)
+let syntax = source.buildSyntax(format: Format())
 
 var text = ""
 syntax.write(to: &text)
-print(text)
 ```
 
 prints:
 
 ```swift
-import SwiftSyntax
-struct ExampleStruct {
-  let syntax: Syntax
+import Foundation
+import UIKit
+class SomeViewController{
+    let tableView: UITableView
 }
 ```
+
+## Usage
+
+### Declare SwiftPM dependency with release tag
+
+Add this repository to the `Package.swift` manifest of your project:
+
+```swift
+// swift-tools-version:5.3
+import PackageDescription
+
+let package = Package(
+  name: "MyTool",
+  dependencies: [
+    .package(name: "SwiftSyntax", url: "https://github.com/apple/swift-syntax.git", .exact("<#Specify Release tag#>")),
+  ],
+  targets: [
+    .target(name: "MyTool", dependencies: ["SwiftSyntaxBuilder"]),
+  ]
+)
+```
+
+Replace `<#Specify Release tag#>` by the version of SwiftSyntaxBuilder that you want to use. see the following [table](https://github.com/apple/swift-syntax#declare-swiftpm-dependency-with-release-tag) for mapping details.
+
+Then, import `SwiftSyntaxBuilder` in your Swift code.
```

Comment:
I think it would be good to also say something about code formatting, like the following

Note that SwiftSyntaxBuilder has limited support for pretty-printing code. We generally suggest running a formatter like [swift-format](https://github.com/apple/swift-format) on the generated code.